client/releases.go

```diff
@@ -29,6 +29,19 @@ func (c *Client) GetReleases(app string) (Releases, error) {
 	return releases, nil
 }
 
+// GetReleasesWithLimit requests releases of an app, overriding the default limit.
+func (c *Client) GetReleasesWithLimit(app string, limit int) (Releases, error) {
```

Comment:
I'm wondering if we should condense this with `GetReleases(app string)` by adding the limit parameter to it. That way we keep one code path when it comes to getting releases.